wallet/src/bin/web_server.rs

```diff
@@ -146,7 +152,7 @@ mod tests {
     }
 
     impl TestServer {
-        async fn new() -> Self {
+        async fn new(path_storage: Option<PathBuf>) -> Self {
```

Comment:
Bit cleaner if we just make this a TempDir, generated in `new()`, instead of passing it in from the test. Not a big deal though

Reply:
Oh yeah good point.  For some reason I was thinking that would mean all test write to the same place, but that's not how TempDir works.  I'll make this change before merging since it's simple enough.